components/plurals/src/provider.rs

```diff
@@ -33,9 +34,3 @@ pub struct PluralRuleStringsV1<'s> {
     pub few: Option<Cow<'s, str>>,
     pub many: Option<Cow<'s, str>>,
 }
-
-icu_provider::impl_data_marker_with_lifetime!(
-    PluralRuleStringsV1<'s>,
-    /// Marker type for [`PluralRuleStringsV1`]
-    PluralRuleStringsV1Marker
-);
```

Comment:
Can you fix the docs for PluralRuleStringsV1Marker so it has a link instead of `#name`?  Here's what it looks like:

![image](https://user-images.githubusercontent.com/1145762/125499254-6b184221-7f57-4ee0-adb3-9aaa241821b5.png)

http://icu4x-pr-artifacts.storage.googleapis.com/gha/672862bbb0bdb8f3c1ed41b91f78165759256499/docs/icu_plurals/provider/struct.PluralRuleStringsV1Marker.html

Reply:
(this probably affects other marker types as well)

Reply:
ugh lol I hoped this would work, guess not